From: Matthias Clasen <mclasen@redhat.com>
Date: Wed, 5 Sep 2018 19:54:32 -0400
Subject: Fix portal path handling

This was broken when I recently introduced this helper
function.

Origin: upstream, commit:47122288b82e50b88e3185d960e2ba5ee8a27a36
Applied-Upstream: 3.24.1
---
 gtk/gtkprivate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkprivate.c b/gtk/gtkprivate.c
index 6428347..a437047 100644
--- a/gtk/gtkprivate.c
+++ b/gtk/gtkprivate.c
@@ -307,7 +307,7 @@ get_portal_path (GDBusConnection  *connection,
     if (sender[i] == '.')
       sender[i] = '_';
 
-  path = g_strconcat ("/org/freedesktop/portal/desktop", "/", kind, "/", sender, "/", token, NULL);
+  path = g_strconcat ("/org/freedesktop/portal/desktop", "/", kind, "/", sender, "/", *token, NULL);
 
   g_free (sender);
 
